### PR TITLE
[EventGrid] Add additional Enum values

### PR DESCRIPTION
**Pull request by mssfang**
Add additional enum values to MediaJobErrorCategroy and MediaJobErrorCode

### CODE REVIEW

File: sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ input-file:
 - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/409af02e5ca217c7e7ec2acf50f4976c053496f8/specification/eventgrid/data-plane/Microsoft.Devices/stable/2018-01-01/IotHub.json
 - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/409af02e5ca217c7e7ec2acf50f4976c053496f8/specification/eventgrid/data-plane/Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json
 - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/409af02e5ca217c7e7ec2acf50f4976c053496f8/specification/eventgrid/data-plane/Microsoft.ServiceBus/stable/2018-01-01/ServiceBus.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/409af02e5ca217c7e7ec2acf50f4976c053496f8/specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json
+- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/6b29b8552671eded065f51ee1b291582ab8cc149/specification/eventgrid/data-plane/Microsoft.Media/stable/2018-01-01/MediaServices.json
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md:84]: We should use the same commit id for all Event Grid swaggers.

File: sdk/eventgrid/microsoft-azure-eventgrid/src/main/java/com/microsoft/azure/eventgrid/models/MediaJobErrorCategory.java [hunk 2]
```diff
@@ -28,7 +28,10 @@
     CONFIGURATION("Configuration"),
 
     /** The error is related to data in the input files. */
-    CONTENT("Content");
+    CONTENT("Content"),
+
+    /** The error is related to account information. */
+    ACCOUNT("Account");
```
Comment (srnagar) [inline sdk/eventgrid/microsoft-azure-eventgrid/src/main/java/com/microsoft/azure/eventgrid/models/MediaJobErrorCategory.java:34]: This is track 1 code. Why are we updating track 1? Are we planning to release Track 1 too?

**Review by srnagar — CHANGES_REQUESTED** [review]

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md [hunk 3]
```diff
@@ -2,13 +2,8 @@
 
 ## 4.10.0-beta.1 (Unreleased)
```
Comment (srnagar) [inline sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:3]: Should the version and date be updated as well or will it be done in a separate PR?

**Review by srnagar — APPROVED** [review]

File: sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md (hunk 3 above)
Comment (mssfang) [reply sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:3]: It will be done in the separate PR

Comment (mssfang) [reply sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md:3]: https://github.com/Azure/azure-sdk-for-java/pull/27568
